tensorflow/core/kernels/mkl_batch_matmul_op.cc

```diff
@@ -314,16 +289,8 @@ class BatchMatMulMkl : public OpKernel {
                           BatchMatMulMkl<CPUDevice, TYPE, true>)
 
 #ifdef ENABLE_MKL
-TF_CALL_float(REGISTER_BATCH_MATMUL_MKL);
-TF_CALL_double(REGISTER_BATCH_MATMUL_MKL);
-TF_CALL_complex64(REGISTER_BATCH_MATMUL_MKL);
-TF_CALL_complex128(REGISTER_BATCH_MATMUL_MKL);
-
 TF_CALL_float(REGISTER_BATCH_MATMUL_MKL_V2);
-TF_CALL_double(REGISTER_BATCH_MATMUL_MKL_V2);
-TF_CALL_complex64(REGISTER_BATCH_MATMUL_MKL_V2);
-TF_CALL_complex128(REGISTER_BATCH_MATMUL_MKL_V2);
-
+TF_CALL_float(REGISTER_BATCH_MATMUL_MKL);
```

Comment:
Nit: Would you mind putting this line before the v2 registration (`TF_CALL_float(REGISTER_BATCH_MATMUL_MKL_V2);`)?